AmplifyPlugins/Core/AWSPluginsCore/Auth/AWSAuthService.swift

```diff
@@ -82,7 +82,7 @@ public class AWSAuthService: AWSAuthServiceBehavior {
     }
     
     /// Retrieves the Cognito token from the AuthCognitoTokensProvider
-    public func getUserPoolAccessToken() async throws -> String {
+    public func getLatestAuthToken() async throws -> String {
```

Comment:
reverting the change in `AWSAuthServiceBehavior` means this will be reverted as well, it should stay as `getUserPoolAccessToken`